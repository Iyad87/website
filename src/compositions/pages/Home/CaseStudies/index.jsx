import React from 'react';

// import Button from '../../../../components/Button';
import Container from '../../../../components/Container';
import TileGrid, { Tile } from '../../../../components/TileGrid';

const BuildBetter = () => (
  <Container wide>
    <TileGrid>
      <Tile title="Trainline">
        A legacy platform that faced a huge pressure to have an updated
        interface.
      </Tile>
      <Tile colour="darker" title="The Economist">
        A legacy platform that faced a huge pressure to have an updated
        interface.
      </Tile>
      <Tile colour="dark" title="Joyent">
        A legacy platform that faced a huge pressure to have an updated
        interface.
      </Tile>
      <Tile title="DoctorLink">
        A legacy platform that faced a huge pressure to have an updated
        interface.
      </Tile>
    </TileGrid>
  </Container>
);

export default BuildBetter;
