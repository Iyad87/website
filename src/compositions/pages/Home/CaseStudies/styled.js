import styled from 'styled-components';
import { spacing } from 'styles/spacing';

export const CallToAction = styled.p`
  margin-bottom: 0;
  margin-top: ${spacing(8)};
  text-align: center;
`;

export const TileContent = styled.div`
  bottom: 0;
  left: 0;
  padding: ${spacing(4)};
  position: absolute;
`;

export const TileContentTitle = styled.h5`
  font-weight: 500;
  font-size: 24px;
  margin: 0;
`;

export const TileContentBody = styled.p`
  font-weight: 300;
  font-size: 24px;
  margin: 0;
`;
