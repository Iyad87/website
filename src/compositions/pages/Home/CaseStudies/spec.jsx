import React from 'react';
import { shallow } from 'enzyme';
import CaseStudies from './index';

test('compositions/homepage/CaseStudies snapshot', () => {
  const wrapper = shallow(<CaseStudies />);

  expect(wrapper).toMatchSnapshot();
});
