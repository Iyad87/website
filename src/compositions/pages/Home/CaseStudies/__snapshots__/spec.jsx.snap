// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`compositions/homepage/CaseStudies snapshot 1`] = `
<styled.div
  wide={true}
>
  <styled.ul>
    <Tile
      colour="normal"
      title="Trainline"
    >
      A legacy platform that faced a huge pressure to have an updated interface.
    </Tile>
    <Tile
      colour="darker"
      title="The Economist"
    >
      A legacy platform that faced a huge pressure to have an updated interface.
    </Tile>
    <Tile
      colour="dark"
      title="Joyent"
    >
      A legacy platform that faced a huge pressure to have an updated interface.
    </Tile>
    <Tile
      colour="normal"
      title="DoctorLink"
    >
      A legacy platform that faced a huge pressure to have an updated interface.
    </Tile>
  </styled.ul>
  <styled.div>
    <Button
      component={[Function]}
      to="/case-studies"
    >
      View More
    </Button>
  </styled.div>
</styled.div>
`;
